study-builder/config/manifest-rgp.rb

```diff
@@ -0,0 +1,3 @@
+render "application.conf.ctmpl"
+render "local.conf.ctmpl"
+render "vars.conf.ctmpl"
```

Comment:
This is the same set of files we render in `manifest.rb`. Do we need this new file?

Reply:
Good point.  I was thinking something wasn't working when I didn't have that file but I think that ended up being a different issue that I solved